docs/rules/README.md

```diff
@@ -1,151 +1,318 @@
 # Rules
 
-Rules in ESLint are divided into several categories to help you better understand their value. All rules are disabled by default. ESLint recommends some rules to catch common problems, and you can use these recommended rules by including `extends: "eslint:recommended"` in your configuration file. The rules that will be enabled when you inherit from `eslint:recommended` are indicated below as "(recommended)". For more information on how to configure rules and use `extends`, please see the [configuration documentation](../user-guide/configuring.md).
+## Overview
 
-Some rules are fixable using the `--fix` command line flag. Those rules are marked as "(fixable)" below. Currently these fixes are mostly limited only to whitespace fixes.
+Rules in ESLint are divided into several categories to help you better
+understand their value, with the rules being introduced below roughly
+according to a descending benefit-cost value, and subgrouped according
+to whether the ratio is due more to benefit or cost, and according to
+whether the rule applies only in specific environments.
 
-## Possible Errors
+All rules are disabled by default.
```

Comment:
:-1: Most people are coming to this page for the list of rules and skip over the text at the beginning. We need the text to be as short as possible and put the focus on the rules list. I think this adds way too much text up front and the more verbose grouping is a bit too much.